src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/Country.java

```diff
@@ -1,10 +1,5 @@
 package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
```

Comment:
I think Country enums can now be deleted.

Reply:
same for UkCountry enum as well

Reply:
Deleted the enums.